### ISSUE TITLE
Will not work with materialize 0.97.1 package

### ISSUE THREAD

**Issue by pfafman** [MeteorStuff/meteor-materialize-modal-test#1]
The site will not load if you use materialize:materialize 0.97.1.

```
Uncaught TypeError: Cannot set property 'guid' of undefined
```

The materialize modal will work if you include materialize not via a package.

**Comment by msolters** [comment]
This is a problem (almost a month old!) with Materialize itself: https://github.com/Dogfalo/materialize/issues/2037

You can easily fix it by locking the package to the last version, which _is_ stable.

```
meteor add materialize:materialize@=0.97.0
```

**Comment by msolters** [comment]
In fact I have already done so, in this case:
https://github.com/MeteorStuff/meteor-materialize-modal-test/blob/master/.meteor/packages#L24

**Comment by pfafman** [comment]
Yeah I know I can lock it was testing with it not locked.  Can you give me permission to update the demo site or just update it from here.  I put a note about form in a form.

**Comment by msolters** [comment]
Done

**Comment by msolters** [comment]
I'd like to point out that the site is currently here, which contains some fixes that I'm not sure are in this fork.  You should consider merging in or manually updating the diffs:  https://github.com/MeteorStuff/meteor-materialize-modal-test/compare/master...msolters:meteorstuffCompatible

Looks like it's really only the capitalization in that URL.  For whatever reason it gives me a 404 if it's not capitalized as I have it in my branch.
